src/util/worker/mod.rs

```diff
@@ -232,13 +217,11 @@ impl<S: Into<String>> Builder<S> {
 
     pub fn create<T: Display>(self) -> Worker<T> {
         let (scheduler, rx) = if self.pending_capacity == usize::MAX {
-            let (tx, rx) = mpsc::channel::<Option<T>>();
-            let sender = TaskSender::Unbounded(tx);
-            (Scheduler::new(self.name, AtomicUsize::new(0), sender), rx)
+            let (tx, rx) = crossbeam_channel::unbounded::<Option<T>>();
+            (Scheduler::new(self.name, AtomicUsize::new(0), tx), rx)
```

Comment:
Looks like this line can be put to L226.

Reply:
Will do.